src/service/packet_router.proto

```diff
@@ -21,6 +26,8 @@ message packet_router_packet_report_v1 {
   bytes payload_hash = 10;
   uint32 payload_size = 11;
   bool free = 12;
+  packet_type type = 13;
+  uint64 hpr_timestamp_ms = 14;
```

Comment:
rename to `receive_timestamp`. In general we don't put the type in the name but put it as a comment above the field.. I.e. 

```
// received timestamp  in ms since unix epoch
uint64 received_timestamp = 14;
```